sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -96,7 +109,23 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     programOffering.parentOffering = programOffering;
 
     try {
-      return await this.repo.save(programOffering);
+      return await this.dataSource.transaction(
```

Comment:
Are we not missing the offering update part ?? pending offerings can be created during offering update as well

![image](https://github.com/bcgov/SIMS/assets/54600590/4f97b006-2714-428f-856d-3d1a3849bd43)

Reply:
It was added now, not executing in parallel, but saving.